resources/js/components/CharLimit.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <div class="progress position-relative">
+      <div class="progress-bar" role="progressbar" v-bind:style="progressBarStyles" :aria-valuenow="percentage" aria-valuemin="0" :aria-valuemax="limit"></div>
+      <span class="justify-content-center align-self-center d-flex position-absolute w-100">{{this.$t('t.views.observations.char_limit')}} <strong class="ml-1"> {{ this.currentValue.length.toFixed(0) }} / {{ limit }}</strong></span>
```

Comment:
Ich würde den Text "Zeichen genutzt" einfach weglassen. Wenn man tippt wird einem glaube ich sehr schnell klar was der Counter bedeutet, und so vermeiden wir das Problem, dass "Character" auf Deutsch keine gute gebräuchliche Übersetzung hat.

Wenn dann würde ich eher ein Argumentarium ala #223 in einem `<help-text>` anzeigen.